Base/res/html/misc/mandelbrot-wasm.html

```diff
@@ -0,0 +1,456 @@
+                        0x0d, 0x20, 0x01, 0x41, 0x02, 0x74, 0x22, 0x0e, 0x20, 0x0d, 0x20, 0x0e,
+                        0x48, 0x1b, 0xb7, 0xa3, 0x22, 0x05, 0xa2, 0x21, 0x07, 0x44, 0x00, 0x00,
+                        0x00, 0x00, 0x00, 0x00, 0xf0, 0x3f, 0x20, 0x02, 0xb8, 0xa3, 0x21, 0x08,
+                        0x41, 0x00, 0x21, 0x0d, 0x03, 0x40, 0x20, 0x01, 0x20, 0x0d, 0x4b, 0x04,
+                        0x40, 0x20, 0x0d, 0xb8, 0x20, 0x06, 0xa1, 0x20, 0x05, 0xa2, 0x21, 0x09,
+                        0x20, 0x00, 0x20, 0x0d, 0x6c, 0x41, 0x01, 0x74, 0x21, 0x10, 0x41, 0x00,
+                        0x21, 0x0e, 0x03, 0x40, 0x20, 0x00, 0x20, 0x0e, 0x4b, 0x04, 0x40, 0x20,
+                        0x0e, 0xb8, 0x20, 0x05, 0xa2, 0x20, 0x07, 0xa1, 0x21, 0x0a, 0x44, 0x00,
+                        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x21, 0x03, 0x44, 0x00, 0x00,
+                        0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x21, 0x04, 0x41, 0x00, 0x21, 0x0f,
+                        0x03, 0x40, 0x20, 0x03, 0x20, 0x03, 0xa2, 0x22, 0x0b, 0x20, 0x04, 0x20,
+                        0x04, 0xa2, 0x22, 0x0c, 0xa0, 0x44, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
+                        0x10, 0x40, 0x65, 0x04, 0x40, 0x02, 0x40, 0x20, 0x03, 0x20, 0x03, 0xa0,
+                        0x20, 0x04, 0xa2, 0x20, 0x09, 0xa0, 0x21, 0x04, 0x20, 0x0b, 0x20, 0x0c,
+                        0xa1, 0x20, 0x0a, 0xa0, 0x21, 0x03, 0x20, 0x02, 0x20, 0x0f, 0x4d, 0x0d,
+                        0x00, 0x20, 0x0f, 0x41, 0x01, 0x6a, 0x21, 0x0f, 0x0c, 0x02, 0x0b, 0x0b,
+                        0x0b, 0x20, 0x10, 0x20, 0x0e, 0x41, 0x01, 0x74, 0x6a, 0x20, 0x03, 0x20,
+                        0x03, 0xa2, 0x20, 0x04, 0x20, 0x04, 0xa2, 0xa0, 0x22, 0x03, 0x44, 0x00,
+                        0x00, 0x00, 0x00, 0x00, 0x00, 0xf0, 0x3f, 0x64, 0x04, 0x7f, 0x20, 0x0f,
+                        0x41, 0x01, 0x6a, 0xb8, 0x20, 0x03, 0x10, 0x00, 0x44, 0x00, 0x00, 0x00,
+                        0x00, 0x00, 0x00, 0xe0, 0x3f, 0xa2, 0x10, 0x01, 0xa1, 0x20, 0x08, 0xa2,
+                        0x44, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xa5, 0x44, 0x00,
+                        0x00, 0x00, 0x00, 0x00, 0x00, 0xf0, 0x3f, 0xa4, 0x44, 0x00, 0x00, 0x00,
+                        0x00, 0x00, 0xfc, 0x9f, 0x40, 0xa2, 0xab, 0x05, 0x41, 0xff, 0x0f, 0x0b,
+                        0x3b, 0x01, 0x00, 0x20, 0x0e, 0x41, 0x01, 0x6a, 0x21, 0x0e, 0x0c, 0x01,
+                        0x0b, 0x0b, 0x20, 0x0d, 0x41, 0x01, 0x6a, 0x21, 0x0d, 0x0c, 0x01, 0x0b,
+                        0x0b, 0x0b, 0x07, 0x00, 0x20, 0x00, 0x2f, 0x01, 0x00, 0x0b, 0x00, 0x17,
+                        0x10, 0x73, 0x6f, 0x75, 0x72, 0x63, 0x65, 0x4d, 0x61, 0x70, 0x70, 0x69,
+                        0x6e, 0x67, 0x55, 0x52, 0x4c, 0x05, 0x66, 0x61, 0x6c, 0x73, 0x65,
+                    ])
+                );
+            })
+                .then(bytes => {
+                    return WebAssembly.compile((bytess = bytes));
```

Comment:
wait, what's the purpose of that?

Reply:
Giving me a way to take a look at it lol
There's a "exports = module.exports" below as well

Reply:
I fixed these up to no longer assign to random global variables.